build-system/tasks/bundle-size.js

```diff
@@ -42,22 +42,40 @@ const {green, red, cyan, yellow} = colors;
  * @param {string} bundleSize the new bundle size in 99.99KB format.
  */
 function storeBundleSize(bundleSize) {
-  const commitHash = gitCommitHash();
   if (!process.env.GITHUB_ARTIFACTS_TOKEN) {
```

Comment:
Something I thought of: I think it's worth adding an explicit check here to make sure that this is:
- a `push` build
- running on `ampproject/amphtml` and not a fork
- running against `master`

Reply:
If this is run from a fork, it'll fail due to the missing token. In `pr-check.js` it only calls `gulp bundle-size --store` for push builds on master. Do you still want to add explicit checks? It could make it harder to debug this locally if anything goes wrong later...

Reply:
I added a check for ampproject/amphtml. Skipping the other two checks since they're already checked elsewhere :)

Reply:
The purpose of the check was to prevent someone who has write permissions and a GH token from being able to manually invoke the task with `--store` and inadvertently write an invalid value to the repo.

I don't see a check for Travis push builds in `bundle-size.js`. It's easy enough to add:

```
process.env.TRAVIS && process.env.TRAVIS_EVENT_TYPE == 'push'
```

In addition, I don't think the `master` check is sufficient. IIRC, PR builds from forks also show a branch name of `master` due to how Travis works.

Reply:
Added the above, PTAL

Reply:
The new check LGTM.